distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex.enso

```diff
@@ -483,7 +482,7 @@ build_tokenization_output_from_match pattern match =
     if pattern.group_count == 1 then match.text 0 else
         # Extract the ranges of the spans of all capturing groups
         group_numbers = 1.up_to pattern.group_count
-        spans = group_numbers.map (n-> match.span n) . filter Filter_Condition.Not_Nothing
+        spans = group_numbers.map (n-> match.span n) . filter i->i.is_nothing.not
```

Comment:
- What is the benefit of using function over one of the predefined `Filter_Condition`s?
- It is equal in functionality, I assume.
- But the function approach is unlikely to work in `DB_Column`, right?